tests/test_api_handler.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import asyncio
+from Modules.api_handler import *
+
+
+class APIHandlerTest(unittest.TestCase):
+    handler = APIHandler(hostname="dev.api.fuelrats.com", tls=True)
+
+    def test_connect(self):
+        yield self.handler.connect()
+        self.assertIsNotNone(self.handler._connection)
+
+    def test_disconnect(self):
```

Comment:
These tests should be declared as coroutines and use the `@unit_test` decorator. Also, you'll want to use `await` or `yield from` instead of just yield